mapUtils.js

```diff
@@ -60,6 +60,8 @@ exports.calculateZoom = function(extent, width, height) {
 
 exports.addOverlayDataToStyle = function(style, overlay) {
   return {
+    glyphs: style["glyphs"] || "",
```

Comment:
This is the actual fix. We were failing to include the glyph url in the merged style, so of course loading glyphs failed.